pkg/io/fileWriter.go

```diff
@@ -0,0 +1,18 @@
+package io
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+// MakeDirAndFile creates directory provided in filePath and creates file for write in.
+func MakeDirAndFile(filePath string, creator string) error {
```

Comment:
Looks more like a `MakeDirForFile` to me. And `io` package has a 100% coverage, you just can't go below that here.

Reply:
Sure! =)